draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1460,10 +1462,18 @@ Content-Type: text/plain
    as quoted-string (<xref target="quoted.strings"/>).
 </t>
 <t>
-   Because commas (",") are used as a generic delimiter between members of a
-   list-based field value, they need to be treated with care if they are
-   allowed as data within those members. Typically, list members that might contain a
-   comma are enclosed in a quoted-string.
+   A field's value can contain on or more members, separated by commas (",").
+   Fields that allow multiple members are referred to as "list-based fields."
+   Fields that only anticipate a single member are referred to as "singleton
+   fields."
+</t>
+<t>
+   Because commas are used as a generic delimiter between members, they need
+   to be treated with care if they are allowed as data within a member. This
+   is true for both list-based and singleton fields, as a singleton field
+   might be sent with multiple members erroneously, and a recipient needs to
+   be able to detect this condition. Therefore, members that might contain a
+   comma are typically enclosed in a quoted-string.
 </t>
```

Comment:
"recipient needs to  be able to detect this condition" - that makes it sound as if this is requirement; unfortunately it is not (see, for instance "Location"). So this may need slightly rephrasing.

Reply:
I still don't see the value in requiring specific behavior on invalid messages unless we have identified an interop failure that needs a consistent resolution. Age does not. I'd either take the max value or a sum, or maybe just mark it stale, but it depends on why I have the cache.

Reply:
I changed to 'should be considered stale'; that makes sense. I don't think we can say that they should be added or that the largest value should be used, because it's not clear what the non-conformant upstream implementation is doing.

Reply:
rephrased